packages/design-system/src/hooks/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { useClipboard } from 'native-base'
```

Comment:
Let's rename the file to `clipboard.ts` to keep here all the clipboard-related hooks & avoid a generic name like `utils`.